presto-hive/src/test/java/io/prestosql/plugin/hive/util/TestFSDataInputStreamTail.java

```diff
@@ -39,6 +39,7 @@
 import static org.testng.Assert.assertTrue;
 import static org.testng.Assert.fail;
 
+@Test(singleThreaded = true) // e.g. test methods operate on shared, mutated tempFile
```

Comment:
optional: While you are here, can you please change the code to not to swallow exceptions in `tearDown`?